### ISSUE TITLE
KeyError requests

### ISSUE THREAD

**Issue by leonkoens** [xolphin/xolphin-api-python#5]
When listen all certificates a KeyError is raised. 

```
import xolphin
client = xolphin.Client('username', 'password')
certificates = client.certificate().all()
```

produces

> Traceback (most recent call last):
>   File "generate.py", line 9, in <module>
>     certificates = client.certificate().all()
>   File "virtualenv/local/lib/python2.7/site-packages/xolphin/endpoint/certificate.py", line 18, in all
>     result = Requests(self.client.get('certificates', {'page': result.page + 1}))
>   File "virtualenv/local/lib/python2.7/site-packages/xolphin/responses/requests.py", line 11, in __init__
>     for request in self._embedded['requests']:
> KeyError: 'requests'
>

**Comment by leonkoens** [comment]
This is the output of `pip freeze`

> certifi==2017.7.27.1
> chardet==3.0.4
> future==0.16.0
> idna==2.6
> pkg-resources==0.0.0
> requests==2.18.4
> urllib3==1.22
> xolphin-api==1.5.0

**Comment by XolphinIvo** [comment]
Should be fixed together with issue #4, this is solved in master, created new tag (1.6.0).
